src/generators/addon-verify/index.mjs

```diff
@@ -82,9 +82,12 @@ export default {
 
             await mkdir(join(output, folderName), { recursive: true });
 
-            files.forEach(async ({ name, content }) => {
-              await writeFile(join(output, folderName, name), content);
-            });
+            for await (const file of files) {
```

Comment:
Don't think an await on the for declaration is needed. For by nature is already blocking.